build_tools/github/trigger_hosting.sh

```diff
@@ -31,10 +31,29 @@ else
      BRANCH=$HEAD_BRANCH
 fi
 
-curl --request POST \
+# Circle CI REST API return HTTP response with 202 status code even when the POST requests fail.
+# Hence we add some handling so that errors are reported on GitHub.
```

Comment:
```suggestion
# Hence we add some handling so that errors are reported on GitHub.
# For details see: https://circleci.com/docs/api/v2/index.html#operation/triggerPipeline
```